### PR TITLE
feat(net): add column codeVersion to HelloMessage

### PR DESCRIPTION
**Pull request by 317787106**
**What does this PR do?**

- add column codeVersion to HelloMessage

**Why are these changes required?**

- know more information about peer

**This PR has been tested by:**
- Unit Tests
- Manual Testing

**Follow up**

**Extra details**

### CODE REVIEW

File: framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java [hunk 1]
```diff
@@ -127,17 +129,23 @@ public String toString() {
...
     ByteString address = helloMessage.getAddress();
-    if (address != null && !address.isEmpty()) {
+    if (!address.isEmpty()) {
       builder.append("address:")
               .append(StringUtil.encode58Check(address.toByteArray())).append("\n");
     }
 
     ByteString signature = helloMessage.getSignature();
-    if (signature != null && !signature.isEmpty()) {
+    if (!signature.isEmpty()) {
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: Is this field has some relationship with the codeVersion?

File: framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java [hunk 2]
```diff
@@ -127,17 +129,23 @@ public String toString() {
...
       builder.append("signature:")
               .append(signature.toByteArray().length).append("\n");
     }
 
+    ByteString codeVersion = helloMessage.getCodeVersion();
```
Comment (lurais) [inline framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:143]: Will this codeVersion be null when received a HelloMessage from a node with previous version ?

Comment (317787106) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:143]: Every column in proto has default value. `codeVersion` has default value `byte[0]`, not null.

**Review by ss3344520 — APPROVED** [review]

**Review by jwrct — APPROVED** [review]

**Review by halibobo1205 — APPROVED** [review]

**Review by Nouri11190 — COMMENTED** [review]
fix and push through Tron Project

File: framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java (hunk 1 above)
Comment (317787106) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: > Is this field has some relationship with the codeVersion?

No, only code optimization.

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: 933756

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: Ma ja sam na neki način i u teretani i ja Pa Pa

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: https://t.me/notpixel/app?startapp=f7012815595_s654946https://t.me/notgames_bot/squads?startapp=cmVmPTcwMTI4MTU1OTU=

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: Zakačite stavke ostave kako ne bi istekle nakon 1 časa

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: https://t.me/notgames_bot/squads?startapp=cmVmPTcwMTI4MTU1OTU=0x1cfa5e643a3436d94732f65561f8c86e2ea910be

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: Prevucite stavke iz međuspremnika da biste ih izbrisali

Comment (Milanhe92) [reply framework/src/main/java/org/tron/core/net/message/handshake/HelloMessage.java:138]: 🇦🇸🇦🇩🇦🇩Fg0x1cfa5e643a3436d94732f65561f8c86e2ea910beZakačite stavke ostave kako ne bi istekle nakon 1 časa

**Pull request merged**